src/libs/GrowlNotification/GrowlNotificationContainer.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
```

Comment:
I think the module structure you'll need here is like this:

```
src/libs/GrowlNotification
---- | index.js     // implementation of GrowlNotification component for all platforms
---- | GrowlNotificationContainer
-------- | index.js     // web/desktop/mobile web implementation of GrowlNotificationContainer
-------- | index.native.js     // ios/android implementation of GrowlNotificationContainer
```

Reply:
Changed